java/core/src/java/org/apache/orc/impl/OrcAcidUtils.java

```diff
@@ -68,7 +69,7 @@ public static long getLastFlushLength(FileSystem fs,
     }
   }
 
-  private static final Charset utf8 = Charset.forName("UTF-8");
+  private static final Charset utf8 = StandardCharsets.UTF_8;
```

Comment:
Please make another PR for this file change. We can merge that first.

Reply:
Merged as part of ORC-754